test/unit/api/subscriptions.spec.ts

```diff
@@ -155,6 +192,14 @@ describe('Subscriptions Tests', () => {
         body: 'hi you!',
         kind: 'forum',
       });
+      thread2 = await modelUtils.createThread({
```

Comment:
what is this 2nd thread being used for?

Reply:
to emit a 2nd notification for the routes to test an array > 1

Reply:
where is that being tested? I'm not sure I'm seeing any check for that in the test itself? And an array > 1 of notifications? If that's what you're testing shouldn't there be an `expect(notificationsList.length).to.be.equal(2)`?